scania-digital-design-system/tegel
# test(message): added unit tests

**Describe pull-request**  
Adding unit test files for Message component 

**Solving issue**  
Fixes: [CDEP-2963](https://tegel.atlassian.net/browse/CDEP-2963)

**How to test**  
1. Check do test cases meet description defined in [here.](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=3FUgqD&nav=MTVfezkyRkEzOTlBLTY3MzgtNDQ0Mi05MDhBLThFM0IwNzBCNUVGNn0)
2. Try to run a branch on your local and try to change some prop/test in index files, test should fail.



[CDEP-2963]: https://tegel.atlassian.net/browse/CDEP-2963?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
